src/main/java/Duke.java

```diff
@@ -1,10 +1,322 @@
+                    } else if (s.contains("deadline")) {
+                        if (s.length() <= 9 || !s.contains("/by")) {
+                            throw new DukeException("☹ OOPS!!! The description of a deadline cannot be empty.");
+                        }
+                        String[] strArr = s.split("/by ");
+                        String description = strArr[0].substring(9).trim();
+                        String by = strArr[1];
+                        deadlineCommand(description, by, list);
+                        total++;
+                        continue;
+
+                    } else if (s.contains("event")) {
+                        if (s.length() <= 6 || !s.contains("/at")) {
+                            throw new DukeException("☹ OOPS!!! The description of an event cannot be empty.");
+                        }
+                        String[] strArr = s.split("/at ");
+                        String description = strArr[0].substring(6).trim();
+                        String date = strArr[1];
+                        eventCommand(description, date, list);
+                        total++;
+                        continue;
+                    } else if (s.contains("delete")) {
+                        String[] strArr = s.split(" ");
+                        if(strArr.length == 1) {
+                            throw new DukeException("☹ OOPS!!! I don't know which task to delete.");
+                        }
+                        int idx = Integer.parseInt(strArr[1]) - 1;
+                        deleteCommand(idx, list);
+                        total--;
+                    } else {
+                        throw new DukeException("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+                    }
+                } catch (DukeException ex) {
+                    System.out.println(ex);
+                }
+            }
+        }
+     }
+
+    /**
+     * prints horizontal line
+     */
+    public static void printHorizontalLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     *  Prints bye message in between two horizontal lines.
+     */
+    public static void byeCommand() {
+        Duke.printHorizontalLine();
+        System.out.println("Bye. Hope to see you again soon!");
+        Duke.printHorizontalLine();
+
+    }
+
+    /**
+     *
+     * @param list the list of tasks
+     * @param counter the number of tasks in the list
+     */
+    public static void listCommand(List<Task> list) {
+        System.out.println("Here are the tasks in your list:");
+        for(int i = 0; i < list.size(); i++) {
+            int j = i + 1;
+            Task t = list.get(i);
+            System.out.println((j) + "." + t);
+        }
+    }
+
+    /**
+     *
+     * @param list the list of tasks
+     * @param array String array 
+     */
+    public static void doneCommand(List<Task> list, String[] array) {
```

Comment:
Would it be clearer to give the function a name like excecuteDoneCommand? This applies to other command related functions as well.